integ/scripts/bash/run-e2e-automated.sh

```diff
@@ -16,7 +16,7 @@
 set -euo pipefail
 
 refreshcreds() {
-    echo "Refreshing credentials"
+    echo "${1:-} Refreshing credentials"
```

Comment:
Already merged, but just thinking... it might be better if there were also another message log at the end of the function that provides the expiration time of the credentials. i.e. The value of the [Expiration field](https://docs.aws.amazon.com/STS/latest/APIReference/API_Credentials.html).